tox.ini

```diff
@@ -3,8 +3,8 @@ envlist =
     py36,
     py37,
     py38,
-    flake8,
-    coverage
+    coverage,
+    pre-commit
 
```

Comment:
Right! That's correct. `flake8` does not need to get run any more, but we now want the `pre-commit` env to run.

Nothing to do here.

Reply:
ok, when should that run?

Reply:
All the envs listed in the envlist block are run, when you just type `tox` (after you installed it).

You could select a single env to run via e.g. `tox -e pre-commit` and then only the pre-commit env is running.